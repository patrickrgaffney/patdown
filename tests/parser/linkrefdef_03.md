[Foo bar]:
<my%20url>
'title'