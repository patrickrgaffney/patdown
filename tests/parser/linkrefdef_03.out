LINK_REFERENCE_DEF: [Foo bar]: my%20url 'title'